### PR TITLE
Aviary Environment Building Tutorial

### PR DESCRIPTION
**Pull request by Ryan-Rhys**
Tutorial.

### CODE REVIEW

File: docs/Building Environments in Aviary.ipynb [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+    "    def __init__(\n",
+    "        self,\n",
+    "        problem_id: str,\n",
+    "        problem: str,\n",
+    "        answer: float,\n",
+    "        config: CalculatorEnvConfig | None = None,\n",
+    "    ):\n",
+    "        \"\"\"An environment for solving simple math problems using a calculator tool and a check answer tool.\n",
+    "\n",
+    "        Arguments:\n",
```
Comment (jamesbraza) [inline docs/Building Environments in Aviary.ipynb:80]: [nit] Google Style uses `Args:` not `Arguments:`

File: docs/Building Environments in Aviary.ipynb [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+    "import pandas as pd\n",
+    "\n",
+    "# SEE: https://huggingface.co/datasets/openai/gsm8k\n",
+    "GSM8K_PUBLIC_SOURCE = \"openai/gsm8k\"\n",
+    "\n",
+    "\n",
+    "class GSM8kDataset(TaskDataset):\n",
+    "    \"\"\"A dataset class for GSM8k.\n",
+    "\n",
+    "    Attributes:\n",
```
Comment (jamesbraza) [inline docs/Building Environments in Aviary.ipynb:250]: Same comment from other class here, `Attributes` should go to `__init__` docstring

**Review by jamesbraza — APPROVED** [review]
Nice work Ryan and thanks for helping SId

File: docs/Building Environments in Aviary.ipynb (hunk 1 above)
Comment (Ryan-Rhys) [reply docs/Building Environments in Aviary.ipynb:80]: No idea how that ended up there, I think I copy/pasted from ChatGPT and didn't do due diligence haha

**Pull request merged**